### PR TITLE
PMM-7961 Updated exporter shared

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,7 +53,7 @@ format:           ## Format the code
 	@echo ">> formatting code"
 	@$(GO) fmt $(pkgs)
 
-FILES = $(shell find . -type f -name '*.go'")
+FILES = $(shell find . -type f -name '*.go' -not -path "./vendor/*")
```
Comment (askomorokhov) [inline Makefile:56]: Just wondering - why you exclude the vendor - looks like we do not have such dir anymore?

Comment (percona-csalguero) [reply Makefile:56]: copy/pasted from old Makefile

Comment (percona-csalguero) [reply Makefile]: Removed

**Review by askomorokhov — APPROVED** [review]
😎 👍

**Review by JiriCtvrtka — APPROVED** [review]

**Pull request merged**